### PR TITLE
[Backend] API Testing for admin login #81

### PR DESCRIPTION
**Pull request by kenkirito**
## Issue that this pull request solves

 Closes: # (issue number)

## Types of changes

_Put an `x` in the boxes that apply_

- [ ] Bugfix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [ ] Documentation update (Documentation content changed)
- [ ] Other (please describe): 

## Checklist

_Put an `x` in the boxes that apply_

- [ ] My code follows the style guidelines of this project
- [ ] I have performed a self-review of my own code
- [ ] I have commented my code, particularly in hard-to-understand areas
- [ ] I have made corresponding changes to the documentation
- [ ] My changes generate no new warnings
- [ ] My changes does not break the current system and it passes all the current test cases.

## Screenshots

Please attach the screenshots of the changes made in case of change in user interface

## Other information

Any other information that is important to this pull request

### CODE REVIEW

**Comment by kenkirito** [comment]
@jackfrost13 please check

**Comment by jackfrost13** [comment]
Will complete all the pending PR's over the weekend. inconvenience regretted!

File: postman/Login form [hunk 1]
```diff
@@ -0,0 +1,137 @@
...
+									"\r",
+									"pm.test(\"Response has a token property\", function () {\r",
+									"    var jsonData = pm.response.json();\r",
+									"    pm.expect(jsonData.token).to.be.a('string');\r",
+									"});\r",
+									"\r",
+									"/*signin*/\r",
+									"pm.test(\"Response has a data property\", function () {\r",
+									"    var jsonData = pm.response.json();\r",
+									"    pm.expect(jsonData).to.be.a('object');\r",
```
Comment (jackfrost13) [inline postman/Login form:56]: not needed

File: postman/Login form [hunk 2]
```diff
@@ -0,0 +1,137 @@
...
+									"});\r",
+									"\r",
+									"\r",
+									"pm.test(\"400 invalid request\", function () {\r",
```
Comment (jackfrost13) [inline postman/Login form:60]: How can you have a contradicting test? One for 200 another for bad request

File: postman/Login form [hunk 3]
```diff
@@ -0,0 +1,137 @@
...
+									"\r",
+									"\r",
+									""
+								],
+								"type": "text/javascript"
+							}
+						}
+					],
+					"request": {
+						"auth": {
```
Comment (jackfrost13) [inline postman/Login form:72]: Why do you need auth for login?

**Review by jackfrost13 — CHANGES_REQUESTED** [review]
Please add more sample request to cover all cases for login like failed password, missing body parameters etc

**Comment by jackfrost13** [comment]
you may refer to [this](https://github.com/HITK-TECH-Community/Community-Website/pull/244#issuecomment-758199289) for a getting some ideas

**Comment by jackfrost13** [comment]
I cannot see any further commits!

**Comment by kenkirito** [comment]
i have made into main branch

**Comment by jackfrost13** [comment]
> i have made into main branch. #81

You need to add commits to a PR

**Pull request closed without merging**